docs/installation.md

```diff
@@ -1,9 +1,16 @@
 # Installing Ruff
 
-Ruff is available as [`ruff`](https://pypi.org/project/ruff/) on PyPI:
+Ruff is available as [`ruff`](https://pypi.org/project/ruff/) on PyPI. We recommend installing Ruff with [uv](https://docs.astral.sh/uv/):
```

Comment:
We could also include instructions for installing uv, but it feels like that shouldn't live here.